package.json

```diff
@@ -53,7 +53,7 @@
 		"wd": "^1.11.1"
 	},
 	"scripts": {
-		"postinstall": "patch-package && npm ci --prefix gutenberg && npm run i18n:check-cache && cd jetpack/projects/plugins/jetpack && npx pnpm install",
+		"postinstall": "patch-package && npm ci --prefix gutenberg && npm run i18n:check-cache && cd jetpack/projects/plugins/jetpack && npx pnpm@6 install",
```

Comment:
I wonder if we can automate this a bit (in a follow up PR) by reading the pnpm version from the Jetpack repo:

```
npx pnpm@`node -e "console.log(require('./jetpack/package.json').engines.pnpm);"`
```

Reply:
I think the version calculation might be a bit more complex because the value can be a range ([reference](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines)). In fact, I notice that the current value is `^6.32.3` so we'd need to parse it. In relation to this, I think the [`semver` package](https://www.npmjs.com/package/semver) might help us with this. Specifically, the function `minVersion` returns the minimum version of a range, hence we could use it to calculate the version.

Reply:
I had a similar concern about automating the version but turns out `npx` can handle the ranges (TIL) :

```
> npx pnpm@"^6.32.3 || ~7.0.1" --version
6.32.11
```
seems to handle anything that is allowed in `package.json` configs

I also found a slightly cleaner way to get the pnpm verison:

```
> npx -c 'echo "$npm_package_engines_pnpm"'
```

Reply:
Oh, that's very interesting, I didn't know it accepted ranges (TIL) 😅 . In that case, it really simplifies this approach as we can use the raw value from `package.json`.

> I also found a slightly cleaner way to get the pnpm verison:
> 
> ```
> > npx -c 'echo "$npm_package_engines_pnpm"'
> ```

Awesome 👏.